src/_minimal/components/settings/settings-structure-estimation.ts

```diff
@@ -35,6 +35,19 @@ export const notificationsSection: ConfObj[] = [
     },
     component: SettingsGeneral,
   },
+  {
+    key: 'notificationsSticky',
+    title: () =>
+      `${api.storage.lang('settings_Notifications_Sticky')} ${api.storage.lang(
+        'settings_Notifications',
+      )}`,
+    condition: () => true,
```

Comment:
Not sure if `condition` is optional in this case, or whether I should make it under the same condition as the other settings.